createLog.js

```diff
@@ -7,7 +7,7 @@ function createLog (options = {}) {
   const {
     name,
     level = process.env.LEVEL || 'info',
```

Comment:
this should use `config.log.level` which in production takes `process.env.LOG_LEVEL`

Reply:
moved calls + code around so `createLog` could know about `config` 👍